src/bsoncxx/types/bson_value/view.cpp

```diff
@@ -184,6 +185,48 @@ bool operator!=(const view& lhs, const view& rhs) {
     return !(lhs == rhs);
 }
 
+// Serializer functions
+void view::_to_field(std::string& object_field) const {
```

Comment:
It is considered a bit of an anti-pattern to pass a non-const reference as a method argument.  I would suggest rewriting these to be pointers, so that it's clear these fields will be modified:

```
void view::_to_field(std::string* object_field);
```